apps/users/models.py

```diff
@@ -372,6 +385,9 @@ def generate_confirmationcode(self):
                                                           string.digits, 60))
         return self.confirmationcode
 
+    def set_unusable_password(self):
+        self.password = ''
```

Comment:
Should we do: https://docs.djangoproject.com/en/1.8/ref/contrib/auth/#django.contrib.auth.models.User.set_unusable_password?